website/next.config.js

```diff
@@ -4,7 +4,7 @@ console.log(`Environment: ${process.env.HASHI_ENV}`)
 
 module.exports = withHashicorp({
   defaultLayout: true,
-  transpileModules: ['is-absolute-url', '@hashicorp/react-.*'],
+  transpileModules: ['is-absolute-url'],
```

Comment:
The recent release of `nextjs-scripts` transpiles `@hashicorp/react-.*` modules by default.